hoodie-common/src/main/java/com/uber/hoodie/common/table/log/HoodieLogFormatReader.java

```diff
@@ -72,14 +88,33 @@ public HoodieLogFile getLogFile() {
   }
 
   private HoodieLogBlock readBlock() throws IOException {
-    // 2. Read the block type
-    int ordinal = inputStream.readInt();
-    Preconditions.checkArgument(ordinal < HoodieLogBlockType.values().length,
-        "Invalid block byte ordinal found " + ordinal);
-    HoodieLogBlockType blockType = HoodieLogBlockType.values()[ordinal];
 
-    // 3. Read the size of the block
-    int blocksize = inputStream.readInt();
+    int blocksize = -1; int ordinal = -1;
```

Comment:
All of this complexity can be removed once the archived logs have run over sometime with the new writer format. Then we can possibly just write a tool to migrate old V1 to V2 logformat and kill all this dirty code.